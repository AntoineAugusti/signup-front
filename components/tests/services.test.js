import ReactTestRenderer from 'react-test-renderer'
import Services from '../services'
import RESOURCE_PROVIDERS from '../../mock/data/resource-providers'

const RESOURCE_PROVIDER = RESOURCE_PROVIDERS[0]
const SERVICES = RESOURCE_PROVIDER.scopes[0].services

describe('components | Services', () => {
  describe('render', () => {
    const props = {
      lists: SERVICES
    }
    const renderer = ReactTestRenderer.create(<Services {...props} />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
