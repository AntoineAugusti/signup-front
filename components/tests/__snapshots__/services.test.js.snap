// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | Services render should match snapshot 1`] = `
<div
  className="form__group"
>
  <label>
    Qui utilise ces donn√©es ?
  </label>
  <ul
    className="tag-list"
  >
    <li
      className="tag"
    >
      <a
        href={undefined}
      >
        BourseSCO
      </a>
    </li>
    <li
      className="tag"
    >
      <a
        href={undefined}
      >
        Lyon : Mon compte
      </a>
    </li>
  </ul>
</div>
`;
